CHANGELOG.md

```diff
@@ -219,6 +220,10 @@ The `iox::cxx::expected` has dropped the requirement for `INVALID_STATE`. With t
 `ErrorTypeAdapter` which was necessary for non enum types was also removed. The specialization
 of `ErrorTypeAdapter` for custom types must therefore also be removed in the user code.
 
+The `InvalidIdString` was removed from `ServiceDescription` and the Wildcard string was replaced
```

Comment:
Can you please add a short code snippet with how it was used before and how it is used now.

Reply:
I think we will have the Wildcard usage in the upcoming findService example. Nothing else change for the user,except for the inability to create an invalid ServiceDescription now (which made no sense anyway). Considering this, I am not sure what needs to be shown.